src/app/common/code-block/CodeBlock.vue

```diff
@@ -95,13 +95,26 @@ async function handleCodeBlockRenderEvent({ preElement, codeElement, language, c
   gap: $kui-space-60;
   margin-bottom: $kui-space-60;
 }
+
 // Makes code block actions sticky
-:deep(.k-code-block-actions) {
+:deep(.code-block-actions) {
   position: sticky;
   z-index: 4;
   top: var(--AppHeaderHeight);
 }
+
 :deep(.k-highlighted-code-block) {
   border: none;
 }
```

Comment:
is `k-highlighted-code-block` still in the new KCodeBlock?

Reply:
No. I’ll remove this rule set as it’s covered by the new addition below.